deployments/charts/kuma/templates/cni-daemonset.yaml

```diff
@@ -51,8 +51,27 @@ spec:
       containers:
         - name: install-cni
           {{- if .Values.experimental.cni }}
+          {{- if .Values.experimental.ebpf.enabled }}
+          image: {{ include "kuma.formatImage" (dict "image" .Values.cni.experimental.imageEbpf "root" $) | quote }}
```

Comment:
I don't think we should use `kuma.formatImage` in this place.
It does some defaults from `global` etc. which we don't want to do. I think we should just explicitly do
```
{{- printf "%s/%s:%s" $registry $repo $tag -}}
```

Reply:
ack

Reply:
addressed